test/config.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { MultiTcrConfig, DexPriceEstimatorConfig, TheGraphApiConfig } from 'types/config'
+
+describe('Test config defaults', () => {
+  it('name', () => {
+    expect(CONFIG.name).toEqual('Mesa - Gnosis Protocol DApp')
```

Comment:
@anxolin just realized a problem with these tests.

You are comparing default configs against the global configs, which might have settings overwritten by custom config.
This test will fail and need to be updated on downstream, causing conflicts when syncing with our base branch.

For example, replacing the name in the custom config breaks it:
![screenshot_2020-05-07_14-40-18](https://user-images.githubusercontent.com/43217/81347498-22eec900-9071-11ea-937a-4afa9a987f67.png)

I addressed this by conditionally loading custom config.

Reply:
Good point!